ui/src/app/views/concept-homepage/component.spec.ts

```diff
@@ -93,6 +70,10 @@ describe('ConceptHomepageComponent', () => {
         // This finishes the page reloading.
         updateAndTick(fixture);
       });
+    urlParamsStore.next({
+      ns: WorkspaceStubVariables.DEFAULT_WORKSPACE_NS,
+      wsid: WorkspaceStubVariables.DEFAULT_WORKSPACE_ID
+    });
```

Comment:
Do you need to do this here:
```
currentWorkspaceStore.next({
      ...WorkspacesServiceStub.stubWorkspace(),
      accessLevel: WorkspaceAccessLevel.OWNER,
    });
```

Reply:
This was extraneous data in the stub, the component doesn't actually use it.